frontend/stylesheets/trace_view/_right_menu.less

```
@@ -93,14 +93,20 @@
 
 .markdown-small {
 
-  h1,
-  h2,
-  h3,
-  h4,
-  h5 {
-    font-size: 16px;
-    margin-bottom: 0;
-    color: var(--ant-color-text);
+  // Markdown headings render as <Typography.Title>, so the class has to be repeated
```

Stylelint (17.14.1)

[error] Unexpected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)